Source/Krypton Components/Krypton.Docking/Elements Impl/KryptonDockingFloatingWindow.cs

```diff
@@ -213,7 +213,7 @@ public override void SaveElementToXml(XmlWriter xmlWriter)
         protected override void LoadDockingElement(XmlReader xmlReader, KryptonPageCollection pages)
         {
             // Grab the requested size and location
-            Point location = CommonHelper.StringToPoint(xmlReader.GetAttribute(@"L"));
+            Point location = CommonHelper.StringToPoint(xmlReader.GetAttribute(@"L")!);
```

Comment:
Change `CommonHelper`s to take a null item, and they then return a default if is null

Reply:
Changed as requested.

The line (217) below this one is done similarly
```cs
Size clientSize = CommonHelper.StringToSize(xmlReader.GetAttribute(@"S")!);
```
Now CommonHelper has more of these one-liners that work in a similar manner.
I leave that for now and put CommonHelper on the list to be revisted later on for a closer look.